docs/README.md

```diff
@@ -48,6 +50,18 @@ make livehtml
 make.bat livehtml
 ```
 
+## Running Sphinx Locally with Docker
+
+To generate the documentation as HTML file, use the provided docker command from the project root:
+```bash
+docker run --rm -v $PWD/docs:/docs $(docker build -q -t sphinx -f docs/Dockerfile ./docs) make html
+```
+
+To auto-generate the documentation as HTML file, use the provided docker command from the project root:
```

Comment:
This text is the same as the one above despite the command doing something else. Please adapt this.

Reply:
`auto-generate` instead of `generate` :D but changed it and added the live-reload feature in the description.